aroma/tests/conftest.py

```diff
@@ -1,8 +1,11 @@
+import os.path as op
 import pytest
 
 from nilearn.datasets import fetch_development_fmri
 
 
+import aroma
```

Comment:
```suggestion
import aroma

```

Just needs to be moved up a line.